### ISSUE TITLE
NTR: Scabies skin preparation, positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8096]
New term request Scabies skin preparation, positive (seidel_190411142104): Detection of scabies mites, eggs, or feces with microscopic examination

**Comment by pnrobinson** [comment]
@MickeySegal
given the abundance of tests for scabies, I think it is probably best to have a term such as Posiitve skin scabies test and mention the options here: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10219715/
thoughts?

**Comment by MickeySegal** [comment]
Our finding is specific for "Skin scrapings with microscopy. The diagnosis of scabies is confirmed through microscopic examination of mites, eggs, or feces from obtained scales by skin scraping."

**Comment by pnrobinson** [comment]
@MickeySegal 
The HPO does not explicitly model diagnostic tests, and it seems to a large extent the various available tests lead to results that are medically equivalent, so for now let's go with

Adding new term: 
- ID: HP:6000899
- label: Posiitve skin scabies test
- parents: [Positive skin infectious agent test (HP:0430117)](https://hpo.jax.org/app/browse/term/HP:0430117)
- pmid: PMID:37233429
- def: Identification of scabies by means of an examination of the skin such as microscopic identification of scabies mites, eggs, or feces from obtained scales by skin scraping.
- comment: Human scabies is a neglected tropical disease of the skin that affects all age groups. It is a highly contagious skin infection that presents as generalized pruritus and is caused by the Sarcoptes scabiei (S scabiei) var. hominis mite, which is a constrained ectoparasite that lives in the epidermis. Classic scabies is expressed by erythematous papular eruptions, burrows, and pruritus.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8096
- synonyms:
